app/views/admin/claim-teacher-1.html

```diff
@@ -0,0 +1,178 @@
+
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  DfE Maths and Physics Service Prototype
+{% endblock %}
+
+{% block beforeContent %}
+  {% from "phase-banner/macro.njk" import govukPhaseBanner %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
```

Comment:
We could probably change these to 'beta' or 'prototype' as we're no longer in alpha but it's not a biggy